x-pack/plugins/security_solution/server/lib/telemetry/sender_v2.types.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { TelemetryPluginSetup, TelemetryPluginStart } from '@kbn/telemetry-plugin/server';
+import { type IUsageCounter } from '@kbn/usage-collection-plugin/server/usage_counters/usage_counter';
+import { type TelemetryChannel } from './types';
+import type { ITelemetryReceiver } from './receiver';
+
+/**
+ * This service sends telemetry events to the telemetry service asynchronously. Managing
+ * different configurations per channel and changing them dynamically without restarting
+ * the service is possible.
+ */
+export interface ITelemetryEventsSenderV2 {
```

Comment:
```suggestion
export interface IAsyncTelemetryEventsSender {
```
Instead of v2, would you be willing to name the new events sender something descriptive like `AsyncTelemetryEventsSender`?